txrelayer/txrelayer.go

```diff
@@ -33,6 +33,8 @@ type TxRelayer interface {
 
 var _ TxRelayer = (*TxRelayerImpl)(nil)
 
+const numRetries = 1000
```

Comment:
Move it to the existing constants block on the top of the file (
https://github.com/0xPolygon/polygon-edge/pull/1258/files#diff-c83c8a1e714409dd49d38a1ac6f69af3a22eeee9947090430b37619db121ccf7R14-R18)

Reply:
fixed: https://github.com/0xPolygon/polygon-edge/pull/1258/commits/98b6e5890ea7c06fda67307fd63fe1387ddc36cd